src/App/Repository/PostRepository.php

```diff
@@ -106,6 +128,16 @@ protected function getTable()
         return 'posts';
     }
 
+    protected function fetchFormAttributes($form_ids = [])
+    {
+        $form_ids = is_array($form_ids) ? $form_ids : [ $form_ids ];
+
+        $attrs = $this->form_repo->getAllFormStagesAttributes($form_ids);
+
+        $this->form_attributes_by_form = $attrs->groupBy('form_id');
+        $this->form_attributes_by_key = $attrs->keyBy('key');
```

Comment:
this looks like a typo maybe? (keyBy)

Reply:
ah no, that's a Collection method , it works :+1:

Reply:
That takes the form attribute queries result and creates a new collection indexed by attribute key

Reply:
oh, fancy. Ok !